Makefile

```diff
@@ -49,22 +50,32 @@ patch-post:
 clean:
 	rm -rf $(PACKAGE_PREFIX)
 
-gen:
+mark-paginated:
+	rm -rf ${SPEC_MARK_PAGINATED_DIR}
+	cp -r ${SPEC_PATCHED_DIR} ${SPEC_MARK_PAGINATED_DIR}
+	for operation_spec in $(shell set -x; find ${SPEC_MARK_PAGINATED_DIR}/paths -name '*.yaml' | sort -n); do \
+		script/mark_paginated.py $$operation_spec; \
+	done
+    
+
+gen: validate
 	${OPENAPI_GENERATOR} generate -g go \
 		--package-name ${PACKAGE_MAJOR} \
 		--http-user-agent "${GIT_REPO}/${PACKAGE_VERSION}" \
 		-p packageVersion=${PACKAGE_VERSION} \
 		-p isGoSubmodule=true \
 		-p disallowAdditionalPropertiesIfNotPresent=false \
+		-c /local/config.yaml \
+		--global-property apis \
 		--git-user-id ${GIT_ORG} \
 		--git-repo-id ${GIT_REPO}/${PACKAGE_PREFIX} \
 		-o /local/${PACKAGE_PREFIX}/${PACKAGE_MAJOR} \
-		-i /local/${SPEC_PATCHED_DIR}/${SPEC_ROOT_FILE}
+		-i /local/${SPEC_MARK_PAGINATED_DIR}/${SPEC_ROOT_FILE}
 
-validate:
+validate: mark-paginated
```

Comment:
I think pagination changes to the spec should be applied at `patch` time, or as a separate step run after after `patch` in the `all` task.

Reply:
I think we wouldn't need `SPEC_MARK_PAGINATED_DIR` in this case.

Reply:
@displague Can do, if @ctreatma agrees

Reply:
I agree that marking paginated operations with the existing process would be preferable